lua/ttt2/libraries/orm.lua

```diff
@@ -0,0 +1,219 @@
+---
+-- Object Relational Model
+-- @author Histalek
+
+if SERVER then
+	AddCSLuaFile()
+end
+
+local sql = sql
+
+orm = orm or {}
+
+local ORMMODEL = {}
+
+---
+-- Returns an object relational model according to the specified databasetable. Does nothing if no databasetable with the given name exists.
+-- @param string tableName The name of the table in the database to create a model for.
+-- @param[opt] boolean force If set to `true` the function will not return a cached version of the model.
+-- @return ormmodel The model of the database table.
+-- @realm shared
+function orm.Make(tableName, force)
+	if IsValid(orm[tableName]) and not force then
```

Comment:
I would suggest using another dedicated empty table to act as a cache, to prevent overriding functions etc on the orm table